src/components/markdown-help.jsx

```diff
@@ -0,0 +1,269 @@
+import React from 'react';
+import Markdown from './markdown';
+import simpleAvatar from '../images/simple-avatar.png';
+
+const TalkMarkdownHelp = () =>
+  (<table>
+    <thead>
+      <tr>
+        <th colSpan="2">Zooniverse Hashtags, Users and Subjects</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Add a Hashtag</td>
+        <td>#hashtag</td>
+      </tr>
+      <tr>
+        <td>Mention User</td>
+        <td>@user</td>
+      </tr>
+      <tr>
+        <td>Mention Subject<br />(project Talk)</td>
+        <td>^S<subject_id /><br />e.g. ^S830273</td>
+      </tr>
+      <tr>
+        <td>Mention Subject<br />(any Talk)</td>
+        <td>@owner/project^S<subject_id /><br />e.g. @zooniverse/wildcam-gorongosa^S830273</td>
+      </tr>
+      <tr>
+        <td>Mention<br />Project Teams</td>
+        <td>
+          @admins - mention the project administrators<br />
+          @moderators - mention the project moderators<br />
+          @researchers or @scientists - mention the project researchers<br />
+          @team - mention the entire Zooniverse team
+        </td>
+      </tr>
+      <tr>
+        <th colSpan="2">Adding Emoji to Posts</th>
+      </tr>
+      <tr>
+        <td>Complete<br />Emoji List</td>
+        <td><a href="http://www.emoji-cheat-sheet.com/" rel="noopener noreferrer" target="_blank">use this website for all supported emoji</a></td>
+      </tr>
+    </tbody>
+  </table>);
+
+const MarkdownHelp = ({ title, talk }) => {
+  const bulletedList = `
```

Comment:
I don't see any string interpolation happening? The backticks in javascript are template strings. I think that's your issue and it's being used in both. Remove the backticks and the markdown should be interpreted correctly inline.